native/cocos/bindings/jswrapper/v8/Utils.cpp

```diff
@@ -115,7 +115,8 @@ void jsToSeValue(v8::Isolate *isolate, v8::Local<v8::Value> jsval, Value *v) {
         }
     } else if (jsval->IsString()) {
         v8::String::Utf8Value utf8(isolate, jsval);
-        v->setString(ccstd::string(*utf8));
+        const char *utf8Str = *utf8;
```

Comment:
Remove this temp variable?

Reply:
Yep, v8::String::Utf8Value has `operator*()` which returns `const char*` and `se::Value` has a method that access `const char*`
```c++
void setString(const char *v);
```